src/extension.ts

```diff
@@ -211,7 +212,7 @@ function setupControlView(context: vscode.ExtensionContext) {
 }
 
 function setupDataTreeView(context: vscode.ExtensionContext) {
-    ext.dataPlaneAccounts = [];
+    ext.dataPlaneAccounts = ext.context.globalState.get<DataPlaneAccount[]>(DataPlaneAccountsKey) || [];
```

Comment:
Do we still need in-memory `ext.dataPlaneAccounts` any more?
globalState should be single source of truth.

Reply:
yes, we still need in-memory ext.dataplaneAccounts, since the globalState is work for all vscode extension. 
And ext.dataPlaneAccounts just work for current ext.